src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DeleteDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,180 @@
+             When a DELETE request is made to referenced resource "/ed-fi/schoolYearTypes/{id}"
+             Then it should respond with 409
+              And the response body is
+                  """
+                    {
+                        "detail": "The requested action cannot be performed because this item is referenced by existing GradingPeriod item(s).",
+                        "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
+                        "title": "Dependent Item Exists",
+                        "status": 409,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": []
+                    }
+                  """
+
+        @ignore
+        Scenario: 07 Verify response when deleting a referenced student
+            Given the system has these "schools"
+                  | schoolId | nameOfInstitution | gradeLevels                                                                      | educationOrganizationCategories                                                                                        |
+                  | 4005     | Test school       | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#First Grade"} ] | [ {"educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#school"} ] |
+              And the system has these "students" references
+                  | studentUniqueId | birthDate  | firstName   | lastSurname |
+                  | "987"           | 2017-08-23 | "firstname" | "lastname"  |
+              And the system has these "studentSchoolAssociations"
+                  | entryDate  | schoolReference    | studentReference           | entryGradeLevelDescriptor                        |
+                  | 2021-07-23 | {"schoolId": 4005} | {"studentUniqueId": "987"} | uri://ed-fi.org/GradeLevelDescriptor#First grade |
+             When a DELETE request is made to referenced resource "/ed-fi/students/{id}"
+             Then it should respond with 409
+              And the response body is
+                  """
+                    {
+                        "detail": "The requested action cannot be performed because this item is referenced by existing StudentSchoolAssociation item(s).",
+                        "type": "urn:ed-fi:api:data-conflict:dependent-item-exists",
+                        "title": "Dependent Item Exists",
+                        "status": 409,
+                        "correlationId": null,
+                        "validationErrors": {},
+                        "errors": []
+                    }
+                  """
+
+        @ignore
+        Scenario: 08 Verify response when deleting a student with more than one reference
```

Comment:
Same as above - does not belong in this file.

Reply:
Removed